central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/Masterminds/semver/v3"
```

Comment:
In other parts of the code we use `semver` (not v3). I think we should open a follow-up task to upgrade the other parts to use `v3` or, would it be ok to use the default version and not `v3` here?